cmd/cored/main.go

```diff
@@ -290,6 +290,19 @@ func main() {
 		opts = append(opts, enableMockHSM(db)...)
 		chainlog.Printf(ctx, "Launching as unconfigured Core.")
 		h = core.RunUnconfigured(ctx, db, sdb, *listenAddr, opts...)
+
+		go func() {
+			ticker := time.Tick(5 * time.Second)
+			for {
+				select {
+				case <-ticker:
+					if conf != nil {
```

Comment:
Added this since I thought we would need to exist if config got updated, but you're right — the process will just relaunch so this never executes